ansible/redpanda/cluster/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
Should this be our BSL?

Reply:
https://github.com/redpanda-data/deployment-automation/commit/0dfda6cdf5a8164523fb04652da4cd968bde7b57

Probably not without discussing with Alex. Presumably he started this with the apache license for a reason.